mirage/config.js

```diff
@@ -258,7 +258,14 @@ export default function configure() {
   this.get('/packages/:id/resources', nestedResourceRouteFor('package', 'resources'));
 
   this.get('/resources/:id', ({ resources }, request) => {
-    return resources.find(request.params.id);
+    let resource = resources.find(request.params.id);
+
+    // if the ID is 1, it'll be a custom title
+    if (request.params.id === '1') {
+      resource.update('isTitleCustom', true);
```

Comment:
This looks like it'll get it us into trouble. Could this be handled in the `resource` `serializer`'s `mapResourceAttrs()` instead?

Reply:
I like it. I pushed up a change for that